### DISCUSSION TITLE
BIN Auctions?

### DISCUSSION THREAD

**Discussion by entity** [HypixelDev/PublicAPI#414] [Q&A]
Hi,

If a user buys an auction via a BIN, will this appear on the `auction_ended` endpoint?

Docs don't mention anything of this, it shows an example with `bin: true` but not whether it was actually purchased or just expired.

Thanks

**Comment by builder-247** [comment]
The auctions  have a `buyer` property if they were sold.

**Reply by entity** [reply]
Yeah, wasn’t sure if this would also appear for BIN auctions
